.github/workflows/wait-for-connection-test.yaml

```diff
@@ -8,9 +8,13 @@ on:
   workflow_dispatch:
     inputs:
       halt-for-connection:
-        description: 'Should this invocation wait for a remote connection?'
-        required: false
-        default: '0'
+        description: 'Should this workflow run wait for a remote connection?'
+        type: choice
+        required: true
+        default: 'no'
+        options:
+        - 'yes'
+        - 'no'
```

Comment:
Neat, i thought something like this should have been possible

Reply:
Yep, pretty cool